.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Close stale issues'
```

Comment:
no-response.yml   vs  stale.yml 
see https://github.com/nextcloud/talk-android/pull/1168/files#diff-564029bc456818e4cbe6e9e4ebf8f5761753f232ac68297598c6283cc7c5f7e5

@AndyScherzinger @nickvergessen any thoughts which one to delete?

Reply:
And the other Android App uses stale.yml ;)